comet-accumulo/src/main/java/org/renci/comet/api/WriteScopeApiController.java

```diff
@@ -94,11 +94,26 @@ public WriteScopeApiController(ObjectMapper objectMapper, HttpServletRequest req
             log.error("Accumulo internal error", e1);
             return new ResponseEntity<CometResponse>(HttpStatus.INTERNAL_SERVER_ERROR);
         }
-        catch (AccumuloSecurityException e1) {
-                // Intentionally ignoring the exception
-                e1.printStackTrace();
+        catch(AccumuloSecurityException e) {
```

Comment:
This should be handled in CometOps::writeScope function where a readRow is done before attempting a write. We should ignore this exception if it occurs on a read invoked from a write as this would mean entry does not exist and we want to create an entry.

Reply:
It's already handled in CometOps.